agentliveness/agent.py

```diff
@@ -145,6 +148,28 @@ def _check_cinder(self):
 
         return 0
 
+    def _check_manila(self):
+        manila = manila_client.Client(session=self._get_session())
+        try:
+            for backend in self.CONF.enabled_share_backends:
+                if backend is not None:
+                    host = "{}@{}".format(self.CONF.host, backend)
+                else:
+                    host = self.CONF.host
+                for service in manila.services.list({'host': host}):
+                    if service.state == 'up':
+                        return 0
+                    else:
+                        logger.error("Agent %s is down, commencing suicide", service.host)
+                        return 1
+
+                logger.warning("Agent hostname %s not registered" % host)
```

Comment:
Shouldn't this be `, host)` instead of `% host)`?

Reply:
Yeah, copy&paste from cinder